isis/src/base/objs/Table/Table.cpp

```diff
@@ -145,7 +147,7 @@ namespace Isis {
    * @param tableStr The table string
    * @param fieldDelimiter The delimiter to separate fields with
   */
-  Table::Table(const QString &tableName, const std::string &tableString, const char &fieldDelimiter) {
+  Table::Table(const QString &tableName, const std::string &tableString, const char &fieldDelimiter, const vector<PvlKeyword> tableAttrs) {
```

Comment:
Can we pass the tableAttrs by reference here or is it not possible due to the default value?